vllm/v1/core/kv_cache_manager.py

```diff
@@ -129,6 +129,9 @@ def get_computed_blocks(self,
         if not self.enable_caching:
             # Prefix caching is disabled.
             return KVCacheBlocks.create_empty(), 0
+        # When the request requires prompt logprobs, we skip prefix caching.
+        if request.sampling_params.prompt_logprobs is not None:
```

Comment:
Good catch. Can you combine this along with the above if-condition?

Reply:
Yes. Will do later.

Reply:
Done. PTAL.